consensus/istanbul/backend/backend.go

```diff
@@ -196,7 +196,7 @@ func (sb *Backend) IsProxy() bool {
 }
 
 func (sb *Backend) IsProxiedValidator() bool {
-	return sb.proxyNode != nil
+	return sb.proxyNode != nil && sb.proxyNode.peer != nil
```

Comment:
Are there potential ramifications of having a proxied validator not connected to their proxy act as if they're not proxied? cc @kevjue

Reply:
In general, I think we shouldn't support that case (e.g. a proxied validator automatically convert itself to standalone if it's not peer'ed).    Seems like that be something that an operator sets explicitly.

However, in this case, seems like only the ethstats specific function uses it, so this would work for now.

@diminator - On a related note, I think that IsProxiedValidator() should return whether or not this node is configured to be a proxied validator, instead of it's peer to it's proxy.  I think that there should be another function along the lines of `IsProxiedValidatorConnected` that would do what you are doing here.

Reply:
great point! let's do this in #742